index.ts

```diff
@@ -32,7 +33,18 @@ for (const fp of filePaths) {
     const key = path.parse(fp).name;
 
     const src = fs.readFileSync(fp, { encoding: 'utf-8'});
-    const data = YAML.parse(src);
+    const data = YAML.parse(src) as FeatureData;
+
+    // Compute Baseline high date from low date.
+    if (data.status?.baseline_high_date) {
+        throw new Error(`baseline_high_date is computed and should not be used in source YAML. Remove it from ${key}.yml.`);
+    }
+    if (data.status?.baseline === 'high') {
+        const lowDate = Temporal.PlainDate.from(data.status.baseline_low_date);
+        const highDate = lowDate.add({ months: 30 });
```

Comment:
The behavior is to clamp to the end of the month if the day of the month in `lowDate` is higher than the number of days in the resulting months. This is different from what this code would do:

```js
d=new Date('2020-08-31')
d.setMonth(d.getMonth()+30)
```

That results in 2023-03-03, where Temporal will give us 2023-02-28. Doesn't matter much, but it's a nice property that all things in one month will map to the same future month.